apps/ledger-live-desktop/src/renderer/init.jsx

```diff
@@ -132,6 +133,13 @@ async function init() {
 
   const isMainWindow = remote.getCurrentWindow().name === "MainWindow";
 
+  // TODO hydrate the store with the bridge/cache
+  listCachedCurrencyIds().then(ids => {
```

Comment:
From the code above it looks like `listCachedCurrencyIds` returns the array of ids and not a promise, right ?